### ISSUE TITLE
Move code to live site

### ISSUE THREAD

**Issue by ultimike** [communityinclusion/escoach#57]
This is an offshoot from [task 56](https://github.com/communityinclusion/escoach/issues/56#issuecomment-911827703).

@thinkcollege What is the process for moving the code to the live site?

thanks,  
-mike

**Comment by thinkcollege** [comment]
@ultimike You are working on the devel branch, right?  I will update that and merge it to rack on my local.  When it looks good I will rebase to the live site on the rack branch.
 Any tips on bringing over configuration?  I am still a little shaky on D8/9 configuration procedure.

**Comment by ultimike** [comment]
Yes, we're working on the `devel` branch.

The only configuration that we touched (created) was the following:

```
taxonomy.vocabulary.what.yml
taxonomy.vocabulary.who.yml
taxonomy.vocabulary.where.yml
```

If you haven't been using the configuration system for everything else, then you'll probably lose stuff if you try to run a config-import just to get the things listed above. Probably best to just create them manually. 

Also - once pulled into a new environment, we'll have to run the (small) term migration that populates the who/what/where terms. I can do that for you once everything is pulled.

Sound like a plan?

-mike

**Comment by thinkcollege** [comment]
That's good.  I will start working on that today and let you know if I have questions.

**Comment by thinkcollege** [comment]
Looks like we already imported the config for the Who/What/Where taxonomies.  They are in the live site now.  So, no config needed.

**Comment by thinkcollege** [comment]
@ultimike  I merged the dasboard code into the rack branch.   The dashboard is now running at https://es-coach.org/dashboard.  I guess you will import the taxonomy terms now?

**Comment by ultimike** [comment]
Do I have credentials on that site? I tried logging in with michael@anelloconsulting.com and the same password as http://sp-escoach.communityinclusion.org/  with no luck.

-mike

**Comment by thinkcollege** [comment]
Hi @ultimike someone got overzealous deleting accounts on the live site.  I gave you an admin account on es-coach.org with your anelloconsulting.com email address and password change45678

**Comment by ultimike** [comment]
No luck. "Unrecognized username or password" 😦

**Comment by ultimike** [comment]
I think it's an email address issue - I tried to reset my password: 

> michael@anelloconsulting.com is not recognized as a username or an email address.

-mike

**Comment by thinkcollege** [comment]
sorry, I saved your account on the way out the door to my Labor Day festivities.  When I got back I noticed that there was field error and the form never saved.  Should be good now

**Comment by ultimike** [comment]
@thinkcollege Thanks - I was able to login. Here's what I did on the https://www.es-coach.org site:

1. I updated my password. 
2. I gave the Administrator role the "Administer migrations" permission. 
3. I confirmed there were no who/what/where terms. 
3. I uninstalled then reinstalled the ES Migrate module (the migration wasn't appearing in the migration interface, and this fixed that minor issue).
4. I manually imported the configuration for our custom migration group and migration ("migrate_plus.migration_group.escoach_dashboard.yml" and "migrate_plus.migration.escoach_dashboard_terms.yml") via https://www.es-coach.org/admin/config/development/configuration/single/import 
5. I ran our migration from https://www.es-coach.org/admin/structure/migrate/manage/escoach_dashboard/migrations
6. I confirmed the proper who/what/where terms had been migrated.

I then tested some scenarios on the dashboard and I think we may have overlooked the use case where the user is not part of a team nor has any data. I have opened a new issue to discuss, #60 

-mike
